tests/dashbio_demos/app_sequence_viewer.py

```diff
@@ -69,197 +69,298 @@ def layout():
+                                     )
+                                 ]),
+
+                        html.Br(),
+
+                        html.Div(id='seq-view-info-aa-comp',
+                                 children=[
+                                     html.Span(
+                                         "Amino acid composition",
+                                         className='seq-view-info-element-title'
+                                     ),
+                                     html.Div(
+                                         id='test-selection'
+                                     )
+                                 ]),
+
+                        html.Br(),
+
+                        html.Div(id='seq-view-info-coverage-clicked',
+                                 children=[
+                                     html.Span(
+                                         "Coverage entry clicked",
+                                         className='seq-view-info-element-title'
+                                     ),
+                                     html.Div(
+                                         id='test-coverage-clicked'
+                                     )
+                                 ]),
+
+                        html.Br(),
+
+                        html.Div(id='seq-view-info-mouse-selection',
+                                 children=[
+                                     html.Span(
+                                         "Mouse selection",
+                                         className='seq-view-info-element-title'
+                                     ),
+                                     html.Div(
+                                         id='test-mouse-selection'
+                                     )
+                                 ]),
+
+                        html.Br(),
+
+                        html.Div(id='seq-view-info-subpart-sel', children=[
+                            html.Span(
+                                "Subpart selected",
+                                className='seq-view-info-element-title'
                             ),
-                            href="/assets/sample_data/tubulin.fasta.txt",
-                            download="tubulin.fasta.txt"
-                        )
+                            html.Div(
+                                id='test-subpart-selection'
+                            )
+                        ])
                     ]
-                ),
+                ))
+            ]
+        ),
 
-                html.Div(
-                    id='sequence-viewer-container',
+        html.Div(id='seq-view-control-tabs', children=[
+            dcc.Tabs(id='seq-view-tabs', children=[
+                dcc.Tab(
+                    label='About',
+                    value='what-is',
+                    children=html.Div(className='seq-view-tab', children=[
+                        html.H4('What is Sequence Viewer?'),
```

Comment:
I don't think this renders very well, plus what follows doesn't answer the question in a way that would be syntactical (it doesn't start with "Sequence Viewer is ..." but with an invitation to "Display ..."). So I would remove this level-4 header. See my comment in the general conversation as well.